published/036-2019-apr-2.md

```diff
@@ -0,0 +1,58 @@
+## Welcome to the IPFS Weekly. 👋
+
+The [InterPlanetary File System (IPFS)](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identity. IPFS enables the creation of completely distributed applications. It aims to make the web faster, safer, and more open. Since that’s a pretty large scope, we track development across the ecosystem in this weekly dispatch.
+
+Looking to get involved? Click on some of the links below, see what we’re up to on [GitHub](https://github.com/ipfs), or join us on [IRC](https://riot.im/app/#/room/#ipfs:matrix.org).
+
+Want this update in your inbox? [Subscribe to our weekly newsletter!](https://tinyletter.com/ipfsnewsletter)
+
+Here are some of the highlights since the last IPFS Weekly.
+
+
+## The latest
+
+
+## Meet [TBA Contributor]
+
+Meet this person!
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/categories/articles/).* 
+
++ [Radicle Architecture](http://radicle.xyz/blog/index.html), Radicle Blog, 28 Mar 2019
++ [InterPlanetary File System](https://medium.com/@snehal1798/interplanetary-file-system-1c00196ae52c), 28 Mar 2019
+
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [go-filecoin 0.1.4 is released!](https://filecoin.io/blog/go-filecoin-0.1.4-release/)
+
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [The Ultimate Guide To Understanding The Differences Between PPIO, Filecoin, and Storj](https://medium.com/@ppio/differences-between-ppio-filecoin-storj-20cdf7b3b02e), 27 Mar 2019
```

Comment:
Another week another example: this is pure advertising to PPIO. Again, does anybody check the articles?

Reply:
@koalalorenzo I read this article and I was excited to see an IPFS-affiliated project, Filecoin, being mentioned. 

I apologize for your distaste with the newsletter, but it's a free community resource and no one is forcing you to read it. I would be happy to see any of your suggestions for more suitable content for this week, if you have any.

Reply:
I should apologise, I didn't know you were sharing this kind of content too. I am also sorry that I don't have any _suitable_ content this week. When I will have I will be sure to share it with you!